Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -248,13 +248,12 @@ def oauth_complete(self, code: str) -> None:
         """
 
     # Query Requests
-    def run_query(self, query_params: Dict[str, Any], use_old_endpoint: bool = False) -> Dict[str, Any]:
+    def run_query(self, query_params: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
please add a UT for this ensuring the correct url is used in the different scenatios.